pages/api/longestStreak.ts

```diff
@@ -0,0 +1,126 @@
+import { shuffle } from "lodash";
+import { BlockState } from "../../components/math/longestStreak/MultiplicationBlock";
+import { GameLevel } from "../../redux/longestStreakSlice";
+import { getRndInteger } from "./random";
+
+export type GameBlockState = {
+  text: string;
+  value: number;
+  isProduct: boolean;
+  state: BlockState;
+};
+
+export function getRangeValues(level: GameLevel) {
+  if (level === GameLevel.BEGINNER) {
+    let a = 1
+    let b = 10
+    console.log(a, b)
+    return [a, b];
+} else if (level === GameLevel.BEGINNER_ADVANCED) {
+    let a = 10
+    let b = 20
+    return [a, b];
+  } else if (level === GameLevel.INTERMEDIATE) {
+    let a = 20
+    let b = 30
+    return [a, b];
+  } else if (level === GameLevel.INTERMEDIATE_ADVANCED) {
+    let a = 30
+    let b = 40
+    return [a, b];
+  } else if (level === GameLevel.EXPERT) {
+    let a = 40
+    let b = 50
+    return [a, b];
+  };
+}
+
+export function initializeGameState(): GameBlockState[] {
+  let dummyArray: GameBlockState[] = [];
+  for (let i = 0; i <= 19; i++) {
+      let getRangeValuesArray = getRangeValues(GameLevel.BEGINNER)
+      console.log("First range num: " + getRangeValuesArray[0])
```

Comment:
console logs are fine for developing but you should never commit them into main. Anyone can see the console log using the inspect tool.